src/engraving/rendering/score/beamtremololayout.cpp

```diff
@@ -143,6 +143,13 @@ void BeamTremoloLayout::offsetBeamToRemoveCollisions(const BeamBase* item, const
 
             if (innerLine == outerLine) {
                 sameLineException = 1;
+            } else if (chordRests.size() == 3) {
+                const Chord* otherOuterChord = outerChord == firstChordRest ? toChord(lastChordRest) : toChord(firstChordRest);
```

Comment:
It seems that this causes https://github.com/musescore/MuseScore/issues/24478#issuecomment-2332310491, because `firstChordRest` is apparently not guaranteed to be a chord.

(And perhaps also causes https://github.com/musescore/MuseScore/issues/24492, who knows...)